garrysmod/lua/includes/extensions/net.lua

```diff
@@ -76,24 +76,14 @@ end
 --
 -- Read/Write a player to the stream
 --
-function net.WritePlayer( ply )
-
-	if ( !IsValid( ply ) || !ply:IsPlayer() ) then 
-		net.WriteUInt( 0, 8 )
-	else
-		net.WriteUInt( ply:EntIndex(), 8 )
-	end
+local maxplayers_bits = math.ceil(math.log(1 + game.MaxPlayers()) / math.log(2))
```

Comment:
Please use spaces around operators and inside brackets (like in contributing.md)
```lua
if ( expression1 == expression2 ) then myVariable = ( 1 + 2 ) / 4 end
```